### PR TITLE
Add styling to TopBar and more unit tests.

### PR DESCRIPTION
**Pull request by burkegg**

### CODE REVIEW

File: client/ReviewModule.jsx [hunk 1]
```diff
@@ -76,16 +77,19 @@ export default class ReviewModule extends React.Component {
...
+  }
+
+  avgStars(reviewsArray) {
     let sum = 0;
-    const { reviews } = this.state;
-    sum = reviews.reduce((acc, curr) => (acc + curr.stars), 0);
-    const avg = sum / reviews.length;
-    this.setState({ avgStars: avg });
+    sum = reviewsArray.reduce((acc, curr) => (acc + curr.stars), 0);
+    const avg = sum / reviewsArray.length;
```
Comment (shaneyuan18) [inline client/ReviewModule.jsx]: Sorry if its kind of late in the game, but I am wondering if you include any edge case for businesses that have no reviews. In those scenarios, this line will break your code.

File: client/TopBar.jsx [hunk 2]
```diff
@@ -1,14 +1,116 @@
...
+`;
 
 export default class TopBar extends React.Component {
   constructor() {
     super();
     this.state = {
-      hovered: 0,
-      grey: [],
-      clicked: false,
+      value: '',
```
Comment (shaneyuan18) [inline client/TopBar.jsx]: I usually only use var name 'value' if I am storing a number. Try to use something like searchString so it's more precise.

File: client/ReviewModule.jsx (hunk 1 above)
Comment (burkegg) [reply client/ReviewModule.jsx]: Good call.

File: client/TopBar.jsx (hunk 2 above)
Comment (burkegg) [reply client/TopBar.jsx]: Done. Thanks.

**Pull request merged**